### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by aruppert**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -39,7 +40,10 @@ function App() {
                   />
                 </Route>
                 <Route>
-                  <HomePage path="/home" />
+                  <HomePage exact path="/home" />
```
Comment (lmachens) [inline src/App.js]: `exact path="/home"` should be at the `<Route exact path="/home">`

File: src/App.js [hunk 2]
```diff
@@ -39,7 +40,10 @@ function App() {
...
+                </Route>
+                <Route>
+                  <ContactPage exact path="/contact" />
```
Comment (lmachens) [inline src/App.js]: `exact path="/contact"` should be at the `<Route exact path="/contact">`

Comment (aruppert) [reply src/App.js]: thx!

File: src/App.js (hunk 1 above)
Comment (aruppert) [reply src/App.js]: thx!

**Pull request merged**